src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -146,6 +161,102 @@ private void validateHearingRequest(UpdateHearingRequest hearingRequest) {
         }
     }
 
+    private GetHearingsResponse getHearingsResponseDetails(String caseRef, List<CaseHearingRequestEntity> entities) {
+        GetHearingsResponse getHearingsResponse = new GetHearingsResponse();
+        if (!entities.isEmpty()) {
+            getHearingsResponse.setCaseRef(entities.get(0).getCaseReference());
```

Comment:
```suggestion
            getHearingsResponse.setCaseRef(caseRef);
```
No need to get this from the entity as the method argument takes it? Then for both the `if`/`else` we would be doing the same, can also take this line above the `if` :)

Reply:
changed